docs/generate-time-series/python.rst

```diff
@@ -0,0 +1,201 @@
+.. _gen-ts-python:
+
+======================================
+Generate time series data using Python
+======================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by sampling your `system load`_ using `Python`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+CrateDB must be :ref:`installed and running <install-run>`.
+
+Make sure you're running an up-to-date Python (we recommend 3.7 or higher).
+
+Then, use `Pip`_ to install the `psutil`_ and  `crate`_ libraries:
+
+.. code-block:: console
+
+    $ pip install psutil crate
+
+
+The rest of this tutorial is designed for Python's `interactive mode`_ so that
+you can experiment with the commands as you see fit. The `standard
+Python interpreter`_ works fine for this, but we recommend `IPython`_ for a more
+user-friendly experience.
+
+
+Use Python to read your system load
+===================================
+
+Start an interactive Python session. Then, import the `psutil`_ library:
+
+    >>>  import psutil
+
+Once imported, you can read the current system load averages with the
+`getloadavg`_ function:
+
+    >>>  psutil.getloadavg()
+    (4.59326171875, 4.56689453125, 5.150390625)
+
+The function returns a tuple with three `floats`_ corresponding to the one
+minute average, five-minute average, and 15-minute average.
+
+
+Set up CrateDB
+==============
+
+Make sure CrateDB running on ``localhost`` with the default configuration.
+
+Then, import the `crate`_ library:
+
+    >>>  from crate import client
+
+`Connect`_ to CrateDB:
+
+    >>>  connection = client.connect()
```

Comment:
Maybe here it would be better to formulate it as `client.connect('localhost:4200')`, otherwise it might not be super helpful if someone wants to do this against a non-local CrateDB (like the cloud, say)